### ISSUE TITLE
Feature request: Set app name

### ISSUE THREAD

**Issue by TheLastProject** [tystuyfzand/gotify-smtp#1]
Hey,

I love this plugin, it works well and is nicely easy to setup. Thank you for that!

I was wondering though, could it be possible to somehow set the app name? I want to couple multiple services that don't natively speak Gotify (Nextcloud, Ofelia, etc) so it would be nice to be able to somehow set the app name. I was thinking of perhaps using something like the source or destination email address (part before @?) to set the app name in Gotify, so that I could make sure the different services would all register as their own app.

**Comment by tystuyfzand** [comment]
I don't think this is possible with the Gotify Plugin API sadly, maybe if they add support for it I could do it!

Currently, the only fields accepted by the SendMessage call are Title, Message, Priority, and Extras. The Application name is set to the Plugin name I think.

**Comment by TheLastProject** [comment]
I've taken a look and indeed couldn't find such an option in the API either. I've created https://github.com/gotify/server/issues/459 to request the Gotify Plugin API being extended to make this possible.
